### PR TITLE
ci(benchmark): add benchmark for tpch

### PR DESCRIPTION
**Pull request by everpcpc**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Summary about this PR

Closes #issue

### CODE REVIEW

File: scripts/benchmark/query/load/tpch.sh [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+SQL
+
+for t in customer lineitem nation orders partsupp part region supplier; do
+  echo "loading into $t ..."
+  cat <<SQL | bendsql query
+COPY INTO $t FROM 's3://repo.databend.rs/tpch10/${t}/'
+credentials=(aws_key_id='$AWS_KEY_ID' aws_secret_key='$AWS_SECRET_KEY') pattern ='${t}.*'
+file_format=(type='CSV' field_delimiter='|' record_delimiter='\\n' skip_header=0);
+SQL
+  cat <<SQL | bendsql query
```
Comment (BohuTANG) [inline scripts/benchmark/query/load/tpch.sh:122]: Should do an `analyze table ` for tables: `customer lineitem nation orders partsupp part region supplier` after load data? @xudong963

Comment (xudong963) [reply scripts/benchmark/query/load/tpch.sh:122]: Yes

**Comment by sundy-li** [comment]
after #10216

**Comment by xudong963** [comment]
> after #10216

Merged @everpcpc

File: benchmark/clickbench/tpch/queries.sql [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+select s_name, s_address from supplier, nation where s_suppkey in ( select ps_suppkey from partsupp where ps_partkey in ( select p_partkey from part where p_name like 'forest%' ) and ps_availqty > ( select 0.5 * sum(l_quantity) from lineitem where l_partkey = ps_partkey and l_suppkey = ps_suppkey and l_shipdate >= to_date('1994-01-01') and l_shipdate < add_years(to_date('1994-01-01'), 1) ) ) and s_nationkey = n_nationkey and n_name = 'CANADA' order by s_name;
+select s_name, truncate(count(*),4) as numwait from supplier, lineitem l1, orders, nation where s_suppkey = l1.l_suppkey and o_orderkey = l1.l_orderkey and o_orderstatus = 'F' and l1.l_receiptdate > l1.l_commitdate and exists ( select * from lineitem l2 where l2.l_orderkey = l1.l_orderkey and l2.l_suppkey <> l1.l_suppkey ) and not exists ( select * from lineitem l3 where l3.l_orderkey = l1.l_orderkey and l3.l_suppkey <> l1.l_suppkey and l3.l_receiptdate > l3.l_commitdate ) and s_nationkey = n_nationkey and n_name = 'SAUDI ARABIA' group by s_name order by numwait desc, s_name;
```
Comment (sundy-li) [inline benchmark/clickbench/tpch/queries.sql:21]: why here use `truncate`?

Comment (sundy-li) [reply benchmark/clickbench/tpch/queries.sql:21]: Oh, It's mistake, but it's trivial

**Review by sundy-li — APPROVED** [review]

**Pull request merged**